<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lumCam</class>
 <widget class="QDialog" name="lumCam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lumiere et Camera</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1161</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Lumiere</string>
    </attribute>
    <widget class="QPushButton" name="ButtonExcel">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>380</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Excel</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>idlum</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Etat de la lumiere :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="laDuree">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>type de la lumiere</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>55</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>duree</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lapiece">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>220</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="idlum">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QComboBox" name="cbEtatlumiere">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Ouverte</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fermee</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QLineEdit" name="leRecherche">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>recherche</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAjouterlumiere">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSupprimerlumiere">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>370</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QTableView" name="tabLumiere">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>771</width>
       <height>311</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonModifererlumiere">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>370</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifer</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbTri">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Tous</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Numéro</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Etat</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dimensions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Matiére</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Déscription</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="buttonSupprimerlumiere_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>370</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>excel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Camera</string>
    </attribute>
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>1161</width>
       <height>481</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="afficherCarteFid">
      <widget class="QTableView" name="tabCarteFid">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>741</width>
         <height>361</height>
        </rect>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_recherche_carteFid">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>40</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>recherche</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_colone_carteFid">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>40</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Tous</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NUM_CARTE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DATE_DE_CREATION</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>POINTS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TOTAL_ACHATS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>STATUT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>REMISE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MATRICULE_FISCALE</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="button_ajouter_carteFid">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_supprimer_carteFid">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>470</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>470</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Retour</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="ajoutClient_2">
      <widget class="QDialogButtonBox" name="validerAjout_cf">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>500</y>
         <width>193</width>
         <height>28</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>651</width>
         <height>431</height>
        </rect>
       </property>
       <property name="title">
        <string>Ajout de carte de fidélité :</string>
       </property>
       <widget class="QSpinBox" name="sb_numCarte">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>50</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="sb_points">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>110</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Num carte :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Points :</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="le_dateCreation">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>50</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>50</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date_de_creation :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Total d'achats :</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="dsb_totalAchats">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>170</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>230</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Statut :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>290</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Remise :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>120</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Matricule fiscale :</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_Status">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>230</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Platinum</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Silver</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gold</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="cb_matriculeFiscale">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>120</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="sb_remise">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>290</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="modifierClient_2">
      <widget class="QDialogButtonBox" name="validerModif_cf">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>500</y>
         <width>193</width>
         <height>28</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
      <widget class="QComboBox" name="cbTriCamera">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>20</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Tous</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Idcam</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Etatcam</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Emplacment</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>351</width>
         <height>351</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_43">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>111</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Numéro de la fenetre :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_44">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Etat de la fenetre :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leDimensionsCamera">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>160</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_45">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Dimensions de la fenetre :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_46">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>300</y>
          <width>55</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QSpinBox" name="spNumeroCamera">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>50</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QComboBox" name="cbEtatCamera">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>100</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Ouverte</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fermee</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QPushButton" name="buttonModifererCamera">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>380</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonSupprimerCamera">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>380</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leRechercheCamera">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>20</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>recherche</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonAjouterCamera">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>380</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QTableView" name="tabCamera">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>60</y>
         <width>471</width>
         <height>291</height>
        </rect>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonSupprimerCamera_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>380</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Excel</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
